mne/viz/utils.py

```diff
@@ -2510,20 +2516,23 @@ def _check_cov(noise_cov, info):
 
 
 def _set_title_multiple_electrodes(title, combine, ch_names, max_chans=6,
-                                   all=False, ch_type=None, ):
+                                   all=False, ch_type=None):
     """Prepare a title string for multiple electrodes."""
     if title is None:
         title = ", ".join(ch_names[:max_chans])
-        if ch_type is not None:
-            ch_type = " " + ch_type[0].upper() + ch_type[1:]
+        ch_type = _channel_type_prettyprint.get(ch_type, ch_type)
```

Comment:
I don't get it.

Reply:
>Is it 'cos we accept keys that are not defined in _channel_type_prettyprint

That's my idea. What if we include fNIRS channels later? If somebody defines a custom channel type at home?

Reply:
I just wanted to know that this was what we were intending. Because from the code is not clear that this is intended or just a side effect.

Reply:
I'd think `your_dict.get(thing, thing)` is how you'd write "give me the value for `thing` if possible, otherwise default to `thing`"?

Reply:
Sorry, keep replying before the edit.

>However I don't like the fact that if a user has a home type this gets treated different. In the sense that it won't have a short name + a pretty print. To make it more explicit, I would pass the dictionary as a parameter (but this adds a parameter that would not be used most of the times). The other option is to add methods to manipulate _channel_type_prettyprint, some sort of registering channels so that users can add custom channels. (But this adds a whole new level of YAGNI functionalities)

This is all internal, so I'm not sure it's that important ..? Right now, if you have None type, it's just "sensors". If you hand-roll your own channel type, if you really want it to work here, you can call it "fNIRS sensor" and it will work perfectly. Or you modify this (private) dict.

The calling function is private too after all, it's only for setting the title for two evoked plotting methods.

Reply:
(yep sorry, somehow today when I try to do a new line I end up submitting the comment)

Reply:
As I said, if, `D.get(k,k)` was intended thats fine with me.